core/src/main/java/com/netease/arctic/catalog/CatalogLoader.java

```diff
@@ -136,6 +138,13 @@ public static ArcticCatalog load(AmsClient client, String catalogName, Map<Strin
             throw new IllegalArgumentException("Internal Catalog support iceberg or mixed-iceberg table only");
           }
 
+          break;
+        case CATALOG_TYPE_GLUE:
+          if (TableFormat.ICEBERG == tableFormat) {
+            catalogImpl = GLUE_CATALOG_IMPL;
+          } else {
+            throw new IllegalArgumentException("Glue Catalog support iceberg table only");
+          }
```

Comment:
MixedIceberg is supported too.

Reply:
I support it, just like the custom catalog.